docs/plugins/utils.md

```diff
@@ -29,19 +29,27 @@ This function is only available in the async context.
 
 Calculate the cached [Url](/docs/plugins/types#shared.url) corresponding to the given file:
 
-- `opts` - Required, the options of the cache, which is a table:
```

Comment:
Any reason for using colons instead of dashes?

Reply:
Not really, I just prefer colons because in the markdown there is already a dash in front.